corehq/apps/reports/tasks.py

```diff
@@ -104,6 +111,75 @@ def get_domains_to_update_es_filter():
     )
 
 
+@periodic_task(run_every=crontab(hour="22", minute="0", day_of_week="*"), queue='background_queue')
+def update_domain_metrics():
+    domains = get_domains_to_update()
+    for chunk in chunked(domains, 5000):
+        update_domain_metrics_for_domains.delay(chunk)
+
+
+@task(queue='background_queue')
+def update_domain_metrics_for_domains(domains):
+    """
+    :param domains: list of domain names
+    """
+    # relying on caching for efficiency
+    all_stats = all_domain_stats()
+    active_users_by_domain = {}
+    for domain in domains:
+        metrics, __ = _update_or_create_domain_metrics(domain, all_stats)
+        if metrics:
+            active_users_by_domain[domain] = metrics['active_mobile_workers']
+
+    datadog_report_user_stats('commcare.active_mobile_workers.count', active_users_by_domain)
```

Comment:
Just wondering out loud, but maybe it would be nice to track more of these metrics in datadog.